mesh/mesh.cpp

```diff
@@ -2154,31 +2152,58 @@ void Mesh::Make3D(int nx, int ny, int nz, Element::Type type,
 #define VTX(XC, YC, ZC) ((XC)+((YC)+(ZC)*(ny+1))*(nx+1))
 
    // Sets elements and the corresponding indices of vertices
-   for (z = 0; z < nz; z++)
+   if (sfc_ordering && type == Element::HEXAHEDRON)
```

Comment:
Potentially, but it would need more work, I think. The tets/wedges making up the "cubes" would have to be ordered too, but I didn't think about it much and wanted to keep it simple for now.